pkg/arvo/app/chat-store.hoon

```diff
@@ -194,8 +194,8 @@
   =:  length.config.u.mailbox  +(length.config.u.mailbox)
       number.envelope.act  length.config.u.mailbox
       envelopes.u.mailbox  (snoc envelopes.u.mailbox envelope.act)
-      inbox  (~(put by inbox) path.act u.mailbox)
   ==
+  =.  inbox  (~(put by inbox) path.act u.mailbox)
   :_  this(inbox inbox)
```

Comment:
Pretty sure the `(inbox inbox)` is redundant.

Reply:
yeah, you're right. will make the change